### PR TITLE
Benchmarks: Configuration - Update inference and network benchmarks in configs

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Update inference and network benchmarks in configs.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: superbench/config/amd_mi100_hpe.yaml [hunk 1]
```diff
@@ -99,9 +99,31 @@ superbench:
         copy_type:
           - sm
           - dma
-    ort-inference:
```
Comment (guoshzhao) [inline superbench/config/amd_mi100_hpe.yaml:102]: can't run on mi100?

**Review by guoshzhao — APPROVED** [review]

File: superbench/config/amd_mi100_hpe.yaml (hunk 1 above)
Comment (yukirora) [reply superbench/config/amd_mi100_hpe.yaml:102]: yes, ort-inference does not register on rocm platform

File: superbench/config/azure_ndmv4.yaml [hunk 2]
```diff
@@ -131,8 +131,19 @@ superbench:
       <<: *default_local_mode
       enable: false
     tensorrt-inference:
-      <<: *default_local_mode
       enable: false
```
Comment (cp5555) [inline superbench/config/azure_ndmv4.yaml:134]: We should enable it.

File: superbench/config/azure_ndmv4.yaml [hunk 3]
```diff
@@ -123,6 +127,44 @@ superbench:
       <<: *default_pytorch_mode
     computation-communication-overlap:
       <<: *default_pytorch_mode
+    ort-inference:
+      <<: *default_local_mode
+      enable: false
```
Comment (cp5555) [inline superbench/config/azure_ndmv4.yaml:132]: We should enable it.

File: superbench/config/azure_ndv4.yaml [hunk 4]
```diff
@@ -112,6 +112,41 @@ superbench:
     ort-inference:
       <<: *default_local_mode
       enable: false
+    tensorrt-inference:
+      enable: false
```
Comment (cp5555) [inline superbench/config/azure_ndv4.yaml:116]: Please enable it.

File: superbench/config/azure_ndv4.yaml [hunk 5]
```diff
@@ -112,6 +112,41 @@ superbench:
...
+      <<: *default_local_mode
+      parameters:
+        pytorch_models:
```
Comment (cp5555) [inline superbench/config/azure_ndv4.yaml:119]: Add precision.

File: superbench/config/azure_ndmv4.yaml (hunk 2 above)
Comment (yukirora) [reply superbench/config/azure_ndmv4.yaml]: ok, updated

File: superbench/config/azure_ndmv4.yaml (hunk 3 above)
Comment (yukirora) [reply superbench/config/azure_ndmv4.yaml]: ok, updated

File: superbench/config/azure_ndv4.yaml (hunk 4 above)
Comment (yukirora) [reply superbench/config/azure_ndv4.yaml:116]: ok, updated

File: superbench/config/azure_ndv4.yaml (hunk 5 above)
Comment (yukirora) [reply superbench/config/azure_ndv4.yaml:119]: added precision int8

**Pull request merged**